about.html

```diff
@@ -0,0 +1,108 @@
+          <li><a class="homepage hidden" href="index.html">Homepage</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li class="campagne"><a href="#">Campagne</a></li>
+        </ul>
+      </nav>
+    </header>
+    <!-- HEADER END -->
+    <!-- HOME -->
+    <section class="home about_home">
+      <h2>"Hello! Sharing world!"</h2>
+      <h1 class="about_home_title">Science Festival Spring 2022</h1>
+      <p class="about_home_text">
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Lorem ipsum dolor sit amet consectetur adipisicing elit. Tenetur itaque mollitia
+        adipisci, voluptate perspiciatis pariatur aliquam laborum.
+        <span class="see_more_text">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore accusantium officiis in nisi reprehenderit vitae illum neque, eius quae
+          porro nulla. Reprehenderit libero nam quaerat autem, quas iure asperiores ad.</span
+        >
+        <br />
+      </p>
+      <p class="home_email">Please contact us per Email for any further questions about Science Festival 2022 !</p>
+      <p class="home_email email">festival2022@science.org</p>
+    </section>
+    <!-- HOME END -->
+
+    <!-- COMMONS -->
+    <section class="commons">
+      <h2>Science Festival Spring 2022 Logo</h2>
+      <hr />
+      <p>The Logo of Science Festival 2022 was decided throught the logo competition from 8. June to 7. July</p>
+      <p class="see_more_text">
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatum consequuntur ducimus illum ullam laudantium consequatur quis expedita
+        veritatis architecto aliquid mollitia commodi nisi enim reprehenderit voluptates non facere, quaerat iste?
+      </p>
+      <img src="assets/images/logo_blanc.png" class="commons_logo" alt="Science Festival Logo" />
+    </section>
+    <!-- COMMONS END -->
+
+    <!-- EVENTS -->
+    <section class="events">
+      <h2>See the past Science Festival events</h2>
+      <hr />
+      <p>Take a look at the last two events which took place in Marseille and Grenoble.</p>
+      <div class="event_list">
+        <div class="event_container">
+          <div class="orange"><span class="date">2019</span><span class="description">Science festival in Marseille</span></div>
+          <img src="assets/images/events/2019.jpg" class="event_image" alt="Image for the 2019 event" />
+        </div>
+        <div class="event_container">
+          <div class="orange"><span class="date">2020</span><span class="description">Science festival in Grenoble</span></div>
+          <img src="assets/images/events/2020.jpg" class="event_image" alt="Image for the 2020 event" />
+        </div>
+      </div>
+    </section>
+    <!-- EVENTS END -->
+
+    <!-- FOOTER -->
+    <footer>
```

Comment:
- [ ] According to the requirements the footer has a white background on mobile size and a dark background on desktop size. Kindly fix it! 🙏

Reply:
it's done
<img width="1229" alt="Capture d’écran 2021-12-21 à 19 34 12" src="https://user-images.githubusercontent.com/7243132/146981224-57cc05be-6c29-488f-87d3-cfd852d83e38.png">
<img width="517" alt="Capture d’écran 2021-12-21 à 19 34 27" src="https://user-images.githubusercontent.com/7243132/146981228-cae6c7eb-b2b3-4906-9c96-d724e67e04b4.png">